clusters/cmgg_clusters.config

```diff
@@ -8,20 +8,17 @@ profiles {
         params {
             config_profile_description  = 'CMGG Slurm config'
             config_profile_contact      = '@matthdsm'
-            max_memory                  = 630.GB
-            max_cpus                    = 75
-            max_time                    = 72.h
         }
         process {
             executor = 'slurm'
             queue = 'debug'
             maxRetries = 2
 
-            withLabel:process_high {
-                cpus   = { check_max( 18    * task.attempt, "cpus"    ) }
-                memory = { check_max( 126.GB * task.attempt, "memory" ) }
-                time   = { check_max( 72.h   * task.attempt, "time"   ) }
-            }
```

Comment:
Maybe keep check_max for now as not all pipelines have been migrated yet, we can remove it when all pipelines use the nextflow options